sdk-setup/src/mer-sdk-chroot

```diff
@@ -396,6 +402,9 @@ setarch i386 chroot ${sdkroot} /bin/su -s /bin/bash -l $user -- -c "$(cat <<END
 export SSH_AUTH_SOCK='${MY_SSH_AUTH_SOCK}'
 export SSH_AGENT_PID=${SSH_AGENT_PID:-}
 export MERSDK='$sdkroot'
+export GNUPGHOME='${HOMEDIR}/${MY_GNUPGHOME_DIR}'
+
+install --directory --mode 700 "\$GNUPGHOME"
```

Comment:
Last nitpick - sorry I didn't realize this earlier - this would be better done from prepare_user(), just after the `if [[ $bind_mount_home == "yes" ]] ; then ... fi` block:

```
install --owner $user --group $(id -gn $user) --mode 700 -d ${sdkroot}${HOMEDIR}/${MY_GNUPGHOME_DIR}
```

(omitted quotes as the surrounding code does :\)